docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+// At some point we need to add a page that talks about how synthetics and
+// RUM/UE work together. Passive/Active, etc.
+
+[[user-experience]]
+= {user-experience}
+
+{user-experience} provides a way to quantify and analyze the perceived performance of your web application.
+Unlike testing environments, {user-experience} data reflects actual users' real-world experiences.
+Powered by the APM Real user monitoring (RUM) agent, all it takes is a few lines of code to begin
+surfacing key user experience metrics and investigative tools.
+Drill down further by looking at data by URL, device type, operating system, browser, and location --
+all of which can have a greater impact on how your application performs on end-users' machines.
+
+[role="screenshot"]
+image::images/user-experience-tab.png[User experience tab]
+
+[discrete]
+[[why-user-experience]]
+== Why is {user-experience} important?
+
+Every website has goals -- some sites want users to buy a product, sign up for a mailing list, download an app,
+or share something on social media.
+But no matter how great your product is, a poor {user-experience} can negatively impact your goal completion rate.
+For example, in one study, 40% of users said they abandon a website if it takes more than three seconds to load.
+footnote:[Source and more info: https://neilpatel.com/blog/loading-time/[neilpatel.com]]
+In another, Amazon calculated that a page load slowdown of just one second would cut conversions by
+7% -- costing them $1.6B in sales each year.
+footnote:[Source and more info: https://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales[fastcompany.com]]
+In short, a good {user-experience} keeps your users happy and improves your website's odds of success.
+
+[discrete]
+[[how-user-experience-works]]
+== How does {user-experience} work?
+
+// I'm not even sure if this section is necessary
+// Some of this is taken from the RUM docs
+
+{user-experience} metrics are powered by the {apm-rum-ref}[APM Real User Monitoring (RUM) agent].
+The RUM agent is installed as a dependency to your application,
```

Comment:
@vigneshshanmugam can we say "or with a few lines of javascript"? Make it seem more like the competitors in terms of ease of instrumentation?